src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1055,30 +1068,38 @@ public override Task<object> ExecuteScalarAsync(CancellationToken cancellationTo
         /// Unlike the ADO.NET method which it replaces, this method returns a Npgsql-specific
         /// DataReader.
         /// </remarks>
-        /// <returns>A DbDataReader object.</returns>
-        public new NpgsqlDataReader ExecuteReader(CommandBehavior behavior) => (NpgsqlDataReader) base.ExecuteReader(behavior);
+        /// <returns>A task representing the operation.</returns>
+        public new NpgsqlDataReader ExecuteReader(CommandBehavior behavior)
```

Comment:
Why not introduce default here too and remove the parameterless overload above?

Reply:
Done.